uint32 x
uint32 y
uint32 size
bool   detected
float64 pitch
float64 roll
float64 yaw_velocity
float64 z_velocity